cli/template/base/.eslintrc.json

```diff
@@ -1,4 +1,22 @@
 {
+  "overrides": [
+    {
+      "extends": [
+        "plugin:@typescript-eslint/recommended-requiring-type-checking"
+      ],
+      "files": ["*.ts", "*.tsx"],
+      "parserOptions": {
+        "project": "tsconfig.json"
+      },
+      "rules": {
+        // Consider enabling these rules if you'd like a bit more strict linting
```

Comment:
I would be ok with having all of these on by default, but I guess we need to decide how strict we want our defaults to be. What does everyone else think?

Reply:
I'd be ok having stricter defaults since this is a new project template. Disabling rules would be ok for me if I have to later on.